source/api/image/3.0/change-log.md

```diff
@@ -30,74 +30,103 @@ The following changes are backwards incompatible with version 2.1.1 of the Image
 
-#### 1.2.6. Change the `profile` property to take one compliance level
+#### 1.2.5. Change the `profile` property to take one compliance level
+
+The `profile` property must have a single value that is a compliance level string. The property value must not be an array as in previous versions, and features supported beyond those specified are instead described in the new `extraFeatures` property.
 
-The `profile` property must have a single value that is a compliance level string. The property value must not be an array as in previous versions, and features supported beyond those specified are instead described in the new `extraFeatures` property. See issues [#1275](https://github.com/IIIF/api/issues/1275), [#1373](https://github.com/IIIF/api/issues/1373), [#1554](https://github.com/IIIF/api/issues/1554), and [#1013](https://github.com/IIIF/api/issues/1013).
+See issues [#1275](https://github.com/IIIF/api/issues/1275), [#1373](https://github.com/IIIF/api/issues/1373), [#1554](https://github.com/IIIF/api/issues/1554), and [#1013](https://github.com/IIIF/api/issues/1013).
 
-#### 1.2.7. Change the `service` property value to an array of objects
+#### 1.2.6. Change the `service` property value to an array of objects
 
-In version 2.1 the value of the `service` property could be a single value or an array of objects, the value must now be an array of objects. See issue [#1131](https://github.com/IIIF/api/issues/1131).
+In version 2.1 the value of the `service` property could be a single value or an array of objects, the value must now be an array of objects.
 
-#### 1.2.8. Remove feature names `sizeByWhListed` and `sizeByForcedWh`
+See issue [#1131](https://github.com/IIIF/api/issues/1131).
+
+#### 1.2.7. Remove feature names `sizeByWhListed` and `sizeByForcedWh`
 
 Per the [deprecation warning][image21-dep-sizes] in the previous version, the feature names `sizeByWhListed` and `sizeByForcedWh` have been removed. See PR [#727](https://github.com/IIIF/api/pull/727).
 
-#### 1.2.9. Remove feature name `sizeByDistortedWh`
+#### 1.2.8. Remove feature name `sizeByDistortedWh`
+
+The feature `sizeByWhDistorted` has no useful meaning separate from `sizeByWh` and was thus removed.
 
-The feature `sizeByWhDistorted` has no useful meaning separate from `sizeByWh` and was thus removed. See issue [#879](https://github.com/IIIF/api/issues/879).
+See issue [#879](https://github.com/IIIF/api/issues/879).
+
+#### 1.2.9. Language map pattern for `label`
+
+In the [Presentation API][prezi3] and here, a new pattern has been adopted for all textual values of a JSON object with the language code as the key (or `@none` if the language is not known) and the content as a string within an array as the value. This applies only to the `label` property in the Image API. The pattern relies on features that have been introduced by the JSON-LD Community Group, and are not yet standardized.
```

Comment:
`none` not `@none` ... do we also need to add mention with `@id` and `@type` (?) .. (it is in https://github.com/IIIF/api/blob/rc2_image_changelog/source/api/image/3/context.json )

Reply:
From the point of view of 2.1 both `@none` and the alias to `none` in the context are new.  My inclination would be to mention it here only, since it's the language map pattern that is the change.